src/github3/gists/history.py

```diff
@@ -43,24 +43,24 @@ class GistHistory(models.GitHubCore):
         The number of deletions from the gist compared to the previous
         revision.
 
-    .. attribute:: totoal
+    .. attribute:: total
 
         The total number of changes to the gist compared to the previous
         revision.
     """
 
-    def _update_attributes(self, history):
+    def _update_attributes(self, history) -> None:
         self.url = self._api = history["url"]
         self.version = history["version"]
         self.user = users.ShortUser(history["user"], self)
-        self.change_status = history["change_status"]
-        self.additions = self.change_status["additions"]
-        self.deletions = self.change_status["deletions"]
-        self.total = self.change_status["total"]
-        self.committed_at = self._strptime(history["committed_at"])
-
-    def _repr(self):
-        return "<Gist History [{0}]>".format(self.version)
+        self.change_status = history.get("change_status")
+        self.additions = self.change_status.get("additions")
+        self.deletions = self.change_status.get("deletions")
+        self.total = self.change_status.get("total")
+        self.committed_at = self._strptime(history.get("committed_at"))
+
```

Comment:
This now makes it indistinguishable between GitHub returning `null` and the key being missing which semantically are two very different things.

Reply:
`get()` should be added then only to additions and deletions right?

Reply:
So missing data makes me think there's a way that GitHub represents the data that is different than the object we have here. (Think about ShortUser versus User versus AuthenticatedUser. There's different representations of the same object with different data. The history object here _seems_ to have another representation. It seems like this happens when retrieving a single `gist` by its id (in the original issue). Given that _used_ to work, that means GitHub's changed something and we probably need a different representation here (sadly).

Reply:
Personally, I'd love some kind of `NotProvided` singleton rather than this many objects but that proved confusing in the past

Reply:
Oh, I rolled back that changes. I am not really sure how to reproduce this other object representation. I don't have much knowledge of Github Gists. Could we use this PR as a hotfix to solve the user's problem and then we dig into this new structure in a new issue?

Reply:
@sigmavirus24 also the user printed self.change_status before the error, the it is not quite another representation:

```
{'total': 2, 'additions': 1, 'deletions': 1}
{'total': 1, 'additions': 1, 'deletions': 0}
{'total': 4, 'additions': 2, 'deletions': 2}
{'total': 2, 'additions': 1, 'deletions': 1}
{'total': 1, 'additions': 1, 'deletions': 0}
{'total': 2, 'additions': 1, 'deletions': 1}
{'total': 3, 'additions': 2, 'deletions': 1}
{'total': 1, 'additions': 1, 'deletions': 0}
{'total': 4, 'additions': 2, 'deletions': 2}
{'total': 1, 'additions': 1, 'deletions': 0}
{}
```